src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/VSTypeScriptBreakpointResolutionResult.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api
+{
+    internal sealed class VSTypeScriptBreakpointResolutionResult
+    {
+        public Document Document { get; }
+        public TextSpan TextSpan { get; }
+        public string LocationNameOpt { get; }
+        public bool IsLineBreakpoint { get; }
+
+        private VSTypeScriptBreakpointResolutionResult(Document document, TextSpan textSpan, string locationNameOpt, bool isLineBreakpoint)
+        {
+            Document = document;
+            TextSpan = textSpan;
+            LocationNameOpt = locationNameOpt;
+            IsLineBreakpoint = isLineBreakpoint;
+        }
```

Comment:
📝 This should be updated to follow the wrapper type pattern (see link to example in other comment).